roles/idm/README.md

```diff
@@ -0,0 +1,12 @@
+#CREATE IDENTITIES IN AN EXISTING IDM 
+
+An ansible role that will consume a google sheet and insert that data into and existing IDM (IPA). the format of the sheet should include a header row
+
+```
+user_name, first_name, last_name, email, expiration_date, group
+```
+
+Only expiration_date is not required. If expiration date is not provide the user's account will never expire (no lockout).
```

Comment:
@mcanoy I think most of these columns are self-explanatory, but maybe we could include an example on the "expiration_date" format - i.e.: is a specific standard needed to be followed for this column?